pkg/kubectl/cmd/create/create_clusterrole.go

```diff
@@ -174,3 +197,22 @@ func (c *CreateClusterRoleOptions) RunCreateRole() error {
 
 	return c.PrintObj(clusterRole)
 }
+
+func generateAggretaionRule(rules []string) (*rbacv1.AggregationRule, error) {
+	matchLabels := map[string]string{}
+	for _, rule := range rules {
+		parts := strings.Split(rule, "=")
+		if len(parts) != 2 {
+			return nil, fmt.Errorf("wrong aggretation rule format.must be like xxx=yyy")
```

Comment:
typo, also i would change it to:
`wrong aggregation rule format. It should be xxx=yyy"